com/win32comext/taskscheduler/test/test_addtask_2.py

```diff
@@ -20,10 +20,10 @@
 
 t = ts.NewWorkItem(task_name)
 t.SetComment("Test a task running as local system acct")
-t.SetApplicationName("c:\\python23\\python.exe")
+t.SetApplicationName("c:\\Python37\\python.exe")
```

Comment:
we should really do the same as we did for test_addtask_1.py here - do you mind opening a followup (either an issue or PR)? Not I'm going to be largely offline for 2 weeks.

Reply:
https://github.com/mhammond/pywin32/pull/2099